index.html

```diff
@@ -59,7 +63,136 @@ <h1 class="info">Hey There. <br>
                </ul>                            
            </div>                   
         </section>           
-    </div>            
+        
+    </div>     
+
+    
+    <div class="wrapper-works">
+        <header class="title">
+            My Recent Works
+        </header>
+
+        <div class="line"></div>
+
+        <div class="image-work">  </div>
+
+        <h3 class="title-multi">
+            Multi-Post Stories
+        </h3>
+
+        <p class="works-description">
+            A daily selection of privately personalized 
+                reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text
+                ever since the 1500s, when an unknown
+                printer took a standard dummy text.
+        </p>
+
+        <ul class="button-container">
+            <li class="button1">css</li>
+            <li class="button1">html</li>
+            <li class="button1">bootstrap</li>
+            <li class="button1">Ruby</li>                
+            <div class="button2">See Project</div>                
```

Comment:
The see project button is a different and separate element from the tags, so please make sure to use the `a` or `button` tag here. And please move it out of the `ul` tag because it's not part of the list.

And it's very important to follow the exact Figma template, so please make sure to update the hover effect colors for your button.

![image](https://user-images.githubusercontent.com/34779928/193323833-4b66a565-4f15-408f-88c6-df2c7040bde4.png)

Reply:
I have made the corrections for the hover background. However, the first issue causes trouble because I put the buttons in a grid with the class button-container. The change proposed affects the proper working of this part.

Reply:
using a 1px solid for border style also introduces a shifting of the boxes and text which is not pleasant, I didn't know how to solve that issue.